src/main/java/edu/ucar/unidata/rosetta/service/FileParserManager.java

```diff
@@ -28,33 +29,16 @@
     public void setParsedFileData(List<List<String>> parsedFileData);
 
     /**
-     * A simple method that reads each line of a file, appends a new line character
-     * and appends to a StringBuffer, and returns the StringBuffer string value. This
-     * method is used to parse the file data when no header lines have been specified.
-     * TODO: refactor to return JSON
+     * A simple method that reads each line of a file, appends a new line
+     * character & adds to a List. The list is then turned into a JSON string.
      *
      * @param filePath The path to the file on disk.
-     * @return A String of the file data parsed by line.
+     * @return A JSON String of the file data parsed by line.
+     * @throws IOException For any file I/O or JSON conversions problems.
      */
-    public String parseByLine(String filePath);
+    public String parseByLine(String filePath) throws IOException;
 
-    /**
-     * This method reads each line of a file and if more than one delimiter has been
-     * specified by the user, it normalizes the delimiters in the line to match the
-     * selectedDelimiter (for ease of parsing purposes on the client-side), or simply
-     * appends  a new line character and appends to a StringBuffer, and returns the
-     * StringBuffer string value. This method is used to parse the file data when
-     * both the header lines and delimiter(s) have been specified by the user.
-     * TODO: refactor to return JSON
-     *
-     * @param filePath          The path to the file on disk.
-     * @param selectedDelimiter The delimiter selected to which any other delimiters will be
-     *                          normalized.
-     * @param delimiterList     The List<String> of delimiters specified by the user.
-     * @param headerLineList    The List<String> of header lines specified by the user.
-     * @return A String of the file data parsed by the delimiter(s).
-     */
-    public String normalizeDelimiters(String filePath, String selectedDelimiter, List<String> delimiterList, List<String> headerLineList);
+    //String normalizeDelimiters(String filePath, String delimiter, List<String> headerLineList) throws IOException;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)